googlemock/src/gmock.cc

```diff
@@ -58,8 +57,7 @@ namespace internal {
 // "=value" part can be omitted.
 //
 // Returns the value of the flag, or NULL if the parsing failed.
-static const char* ParseGoogleMockFlagValue(const char* str,
-                                            const char* flag,
+static const char* ParseGoogleMockFlagValue(const char* str, const char* flag,
```

Comment:
Formatting seems to change without any actual changes. Could you please make sure that only actual changes are included

Reply:
Yes I will fix this.  I think clang-format is ran automatically on the file edit.